utilities/cassandra/format.h

```diff
@@ -165,6 +165,7 @@ class RowValue {
   void Serialize(std::string* dest) const;
   RowValue PurgeTtl(bool* changed) const;
```

Comment:
PurgeTtl/ExpireTtl/Gc might cause confusion. Actually @sagar0 has asked me the difference between them previously. So it might worth add comments decribe the difference between them.

Reply:
Right. It does seem that there's some similarity in all these functions: Purge, Expire, GC and Collectable. And the names could be made better; at least some comments.

Reply:
What about:
PurgeTTL -> RemoveExpiredColumns
ExpireTTL -> ConvertExpiredColumnsToTombstones
GC -> RemoveTombstones

Reply:
yeah, those name definitely convey the meaning much better than the old ones.